recipe/meta.yaml

```diff
@@ -19,14 +19,14 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.7
-    - setuptools_scm
-    - setuptools
+    - python >=3.9
+    - setuptools_scm >=3.4.3
+    - setuptools >=61
   run:
-    - python >=3.7
-    - setuptools
-    - pint >=0.13
-    - pandas >=1.0.4
+    - python >=3.9
+    - pint >=0.21
+    - pandas >=1.5
```

Comment:
While this does match what's in `pyproject.toml`, according to the changelog, it's Pint 0.24 and Pandas 2.2:

https://github.com/hgrecco/pint-pandas/blob/899dd266d6457f672c3b53f44f71e1e91a74ef25/CHANGES#L10-L18

Reply:
Thanks @dopplershift, I missed that.